README.md

```diff
@@ -87,7 +87,7 @@ Here are few points before you start contributing:
 * Do not introduce `utils` or `tools` common packages.
 * Keep the line of sight (happy code path).
 * PostgreSQL version we currently use in production is v14+ so take advantage of all modern SQL features.
-* Use `BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY` for primary keys.
+* Use `BIGINT GENERATED BY DEFAULT AS IDENTITY (START 101) PRIMARY KEY` for primary keys. This allows us to use 1-100 in our seed data (see above).
```

Comment:
I found an issue with our seed script, GENERATED BY allows explicit creation of PKs which we take advantage of there (so you can create meaningful test data), however sequence will start with 1 which will lead to error for the very first INSERT statement, this starts sequence from 101 so we have 100 primary key numbers to work with. it is a change of existing migration which we should not generally do but I would like to do it for this case so its not incorrectly done in the history.

Reply:
I do not like to be a party booper, but I do not love the resolution of that issue by lets just start at random high number. It could easily lead to issues again once we have 100 records and seems like quite hidden issue.

Wouldn't it be easier to just `ALTER SEQUENCE serial RESTART WITH <last seed +1>;` after the seed data? It is easily editable while adding new seeds and clear why it is there :thinking:

Reply:
I considered this too, decided to do the start approach because I felt like it is safer. I implemented a function that resets everything automatically, so no need to worry about this anymore.